### PR TITLE
Updates for comparison of v3.1 new samples to the v2 exomes using pc relate

### PR DESCRIPTION
**Pull request by jkgoodrich**
We realized that there are some samples new to v3.1 (not in v3) where the sample ID didn't match a v2 exome sample ID, but was a duplicate sample based on formal relatedness checks with pc_relate (as was done for the v3 genomes with the v2 exomes). 

This code updates the previous v2 exome/v3 gnome pc_relate code to only compare v3.1 new samples to v2 exomes.

### CODE REVIEW

File: gnomad_qc/v3/sample_qc/v2_pc_relate.py [hunk 1]
```diff
@@ -3,41 +3,69 @@
...
+from gnomad_qc.v2.resources.basics import get_gnomad_meta
 from gnomad_qc.v2.resources.sample_qc import get_liftover_v2_qc_mt
 from gnomad_qc.v3.resources.meta import meta
-from gnomad_qc.v3.resources.sample_qc import (qc, v2_v3_pc_relate_pca_scores,
-                                              v2_v3_relatedness)
+from gnomad_qc.v3.resources.sample_qc import (
+    qc,
+    v2_v3_pc_relate_pca_scores,
+    v2_v3_1_new_pc_relate_pca_scores,
+    v2_v3_relatedness,
```
Comment (klaricch) [inline gnomad_qc/v3/sample_qc/v2_pc_relate.py:13]: do imports have to be alphabetical as well?

File: gnomad_qc/v3/sample_qc/v2_pc_relate.py [hunk 2]
```diff
@@ -3,41 +3,69 @@
...
-        relatedness_ht.write(v2_v3_relatedness.path, args.overwrite)
+        relatedness_ht.write(relatedness_path, args.overwrite)
 
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument('--overwrite', help='Overwrite all data from this subset (default: False)', action='store_true')
     parser.add_argument('--join_qc_mt', help="Join liftover v2 exomes release and v3 release QC MT", action='store_true')
     parser.add_argument('--run_pc_relate', help='Run PC-relate on v2 exomes/v3 combined release QC MT', action='store_true')
+    parser.add_argument('--filter_to_new_3_1_samples', help='Only run the join/PC_relate on v2 exomes combined with v3.1 new samples.', action='store_true')
```
Comment (klaricch) [inline gnomad_qc/v3/sample_qc/v2_pc_relate.py:69]: above help statements don't have a period but this one does

**Review by klaricch — CHANGES_REQUESTED** [review]

**Comment by gtiao** [comment]
Julia says this code is outdated but not worth spending the time to polish, but it may be useful to use as a model for writing code for v4 exomes to v3 genome comparison. Only Sanna has used this code.

**Comment by ch-kr** [comment]
Closing this as it has been linked in another ticket

**Pull request closed without merging**